src/hooks/useSubscriptionPlan.ts

```diff
@@ -0,0 +1,30 @@
+import {useMemo} from 'react';
+import {useOnyx} from 'react-native-onyx';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import {isEmptyObject} from '@src/types/utils/EmptyObject';
+
+function useSubscriptionPlan() {
+    const [policies] = useOnyx(ONYXKEYS.COLLECTION.POLICY);
+
+    // Filter workspaces in which user is the admin and the type is either corporate (control) or team (collect)
+    const adminPolicies = useMemo(
+        () =>
+            Object.values(policies ?? {}).filter(
+                (policy) => policy?.role === CONST.POLICY.ROLE.ADMIN && (CONST.POLICY.TYPE.CORPORATE === policy?.type || CONST.POLICY.TYPE.TEAM === policy?.type),
+            ),
+        [policies],
+    );
```

Comment:
We want to filter to policies where the user is the _owner_, not just an admin. There are `owner` and `ownerAccountID` properties on the policy object we can use. Looks like there's already an `isPolicyOwner` util function too

Reply:
Got it, should we check the role at all? Or is it enough to verify `ownerAccountID`?

Reply:
The current implementation checks the `ownerAccountID` only, let me know if that's enough!